30-remote-site-setup.sh

```diff
@@ -62,7 +62,7 @@ update-all-remote-wn-clients --log-dir /var/log/condor-ce/
 # Populate the bosco override dir from a Git repo
 GIT_SSH_KEY=/etc/osg/git.key
 [[ -f $GIT_SSH_KEY ]] && export GIT_SSH_COMMAND="ssh -i $GIT_SSH_KEY"
-[[ -z $BOSCO_GIT_ENDPOINT ]] || \
+[ "x${BOSCO_GIT_ENDPOINT}" != "x" ] && \
```

Comment:
I don't think you have semantically changed anything here.  The `set -u` at the top is what's causing the error about unset vars.